feature/experimental/tunnel/otg_tests/tunnel_interface_based_resize_test/README.md

```diff
@@ -0,0 +1,97 @@
+3. Configure dualstack IPv4 and an IPv6 addresses for each of the 64xtunnel interfaces. The IP addresses can be of /30 and /127 prefix-lengths.
+4. Pick 2 different /24 IPv4 and /64 IPv6 subnets each to emulate destination of payload traffic to be tunneled. Note: This is for the destination prefixes of the flows generated by OTG:Port1.
+5. Add 128 different v4 and v6 static routes (64 for each address-family) for the 2 sets of /24 IPv4 and /64 IPv6 payload prefixes in bullet#4 above. The static routes should be such that, the first set of 1x /24 IPv4 and 1x /64 IPv6 prefixes should point at the 32x tunnel interfaces configured w/ IPv4 tunnel destination and the second set of 1x /24 IPv4 and 1x /64 IPv6 prefixes should point at the 32x tunnel interfaces w/ IPv6 tunnel destinations.
+
+### Verification:
+ *   Ensure all tunnel interfaces are up
+ *   Ensure all LAG and singleton interfaces are up
+ *   Run Health-1.1. checks
+
+## Subtest-2: Gather baseline stats by passing traffic
+ *   Start with the configuration in Subtest-1 above.
+ *   The test should send 2000 IPv4 and 2000 1Pv6 flows from the ATE:Port1 to the DUT:Port1. These flows should equally be split between the 2x /24 IPv4 and 2x /64 IPv6 destination prefixes mentioned in Subtest-1 bullet#4 above. The flows should use variations in the following,
+        * Source IP address
+        * Source Port
+        * Destination IP adresses.
+        * Destination Port
+        * Payload protocol
+ *   Incoming traffic on DUT:PORT1 should be load shared across all 64 Tunnel interfaces for encapsulation and then routed out the combination of lag and singleton interfaces, i.e. lag1, lag2 and Port6.
+ *   Follow all the "verifications" steps in [*] below
+
+## Subtest-3: Reduce number of tunnel interfaces 
+ *   Start with the state in Subtest-2 above with active traffic flow.
+ *   Reduce the number of Tunnel interfaces, e.g. from 32 to 16 for each of the tunnel destination group by address-family:
+        * If the static routes are used to forward traffic to tunnel, please disable or delete the static routes in this test to simulate the reduction in available paths.
+ *   Incoming traffic on DUT:PORT1 should now be load shared across all remaining 16 Tunnel interfaces per tunnel-destination by address-family.
+ *   Follow all the verifications steps in [*] below
+ *   Inflight reduction in number of tunnel interfaces shouuldn't cause traffic drop.
+
+## Subtest-4: Restore the number of tunnel interfaces 
+ *   Start from the condition in Subtest-3 above with inflight traffic from OTG:Port1.
+ *   Increase number of Tunnel interfaces, e.g. From 16 to 32 per tunnel-destination by address-family.
+ *   Incoming traffic on DUT:PORT1 should now be load shared across all 32 Tunnel interfaces per tunnel-destination by address-family. Follow all the verifications steps in [*] below
+ *   Inflight increase in the number of tunnel interfaces and related static route shouldn’t cause traffic drops.
```

Comment:
Minor nit: Is this redundant to the Verification steps below?

Reply:
Updated, thanks